mimic/model/markov_chain_model.py

```diff
@@ -1,8 +1,57 @@
 """Markov chain model class."""
+
+import random
 from mimic.model.model import Model
+from collections import defaultdict
 
 
 class MarkovChainModel(Model):
-    """ML model utilizing Markov chain."""
+    """A type of model."""
+
+    def __init__(self, stateLength):
+        """
+        Constructor.
+
+        Takes an int stateLength as an argument
+        and instantiates a model.
+        """
+        self.order = stateLength
+        self.groupSize = stateLength + 1
+        self.dict = defaultdict
+        self.data = None
+
+    def learn(self, data):
+        """
+        Learn method.
+
+        Takes in a list of words as an argument
+        and constructs a dictionary based
+        on stateLength provided by the user.
+        """
+        print('Learning...')
```

Comment:
We'll want to make this logging (e.g. with the standard library `logging` module) later - I'll add that as a ticket for now.

Reply:
I will look into it right now!

Reply:
I made a ticket for this (#27) btw.